### DISCUSSION TITLE
Get properties from included model (Typescript)

### DISCUSSION THREAD

**Discussion by TiLopes** [sequelize/sequelize#15864] [Help & Questions]
I'm using the generated models from sequelize-auto. If I want to get the property of an included model it returns undefined or null. However the property exists because if I log the variable.getDataValues() it shows up. If I  set the type of the variable to "any" it works but defeats the whole purpose of using typescript.

Is there a problem in the models associations? Is there another way to get the properties of an included model?

The models and the query:
https://gist.github.com/TiLopes/f342e0258392e4c5dd052baf5013f0da

**Comment by ephys** [comment]
Which version (including minor & patch) of sequelize are you using?

**Reply by TiLopes** [reply]
"sequelize": "^6.29.3"
"sequelize-auto": "^0.8.8"

**Reply by ephys** [reply]
Are you getting a warning about attributes being shadowed?

**Reply by TiLopes** [reply]
I have figured out the problem. Sequelize-auto --noAlias flag does not work as intended or at least as I expected, if I remove that flag and use the necessary 'as' keyword in include it starts working. Should I create an issue in that repo?
